rocky/reports/templates/report_overview/scheduled_reports_table.html

```diff
@@ -77,28 +88,35 @@
                                                         <a href="{% url "view_report" organization.code %}?report_id={{ report.reference }}&observed_at={{ report.observed_at|date:"Y-m-d H:i:s:u" }}"
                                                            title="{% translate "Show report details" %}">{{ report.name }}</a>
                                                     </td>
-                                                    <td>
-                                                        <span class="label tags-color-{{ report.report_type|get_report_type_label_style }}">{{ report.report_type|get_report_type_name }}</span>
-                                                    </td>
                                                     <td>
                                                         {% if report.input_oois|length == 1 %}
                                                             {% with ooi=report.input_oois.0 %}
                                                                 <a href="{% ooi_url 'ooi_detail' ooi organization.code query=ooi.mandatory_fields %}">{{ ooi|human_readable }}</a>
                                                             {% endwith %}
                                                         {% else %}
-                                                            {{ report.input_oois|length }}
+                                                            {{ report.input_oois|length }} {% translate "objects" %}
```

Comment:
Would be nice to use pluralization here:
```
{% blocktrans count report.input_oois as objects %}
    {{ objects }} object
    {% plural %}
    {{ objects }} objects
{% endblocktrans %}
```

Reply:
This can never be "object" since the if-expression already catches the `report.input_oois|length == 1`. So the else is always plural.